pkg/minishift/registration/redhat_test.go

```diff
@@ -75,22 +94,14 @@ func TestRedHatRegistratorRegister(t *testing.T) {
 	}
 	commander := provision.GenericSSHCommander{Driver: d}
 	registrator := NewRedHatRegistrator(commander)
-	s.CommandToOutput["sudo subscription-manager version"] = `server type: RedHat Subscription Management`
-	if err := registrator.Register(param); err != nil {
-		t.Fatal("Distribution shouldn't have to register")
-	} else {
-		cmd := "sudo subscription-manager register --auto-attach --username foo --password foo"
-		if _, ok := s.Commands[cmd]; ok{
-			t.Fatalf("Expected command: %s", cmd)
-		}
-	}
+
 	s.CommandToOutput["sudo subscription-manager version"] = `server type: This system is currently not registered.`
 	if err := registrator.Register(param); err != nil {
 		t.Fatal("Distribution should able to register")
 	} else {
 		cmd := "sudo subscription-manager register --auto-attach --username foo --password foo "
```

Comment:
@hferentschik I think I will remove `cmd` altogether and use `expectedCMDRegistration` and `expectedCMDUnregister` variable for test cases.

```
if _, ok := s.Commands[expectedCMDRegistration]; !ok{
            t.Fatalf("Expected command: %s", expectedCMDRegistration)
}
```